content/updates/2021-12/1-TE-GE.md

```diff
@@ -0,0 +1,18 @@
+---
+pim_announcement_img:
+pim_announcement_alt_img:
+pim_announcement_audience:
+- EE
+- GE
+---
+
+# Tailored Exports available for Growth Edition Users
+::: meta-data type="New" features="Productivity" available="mid-December" in="EE,GE" link-to-doc="../articles/tailored-export.html#discover-operations"
+
+Growth Edition Users can now benefit from customized product and product model export structure, content and sources.
+Operations were also boosted for all users. You can now use symbols (12,28 m²), rounded numbers for measurements, and you can define the number of digits after the separator.
```

Comment:
I'm torn about this being two different announcements vs one. If we're keeping them together we should update the title because otherwise EE customers will just skip right over this one thinking it doesn't apply to them